src/iotjs.c

```diff
@@ -189,6 +190,28 @@ static jerry_value_t dummy_wait_for_client_source_cb() {
   return jerry_create_undefined();
 }
 
+static void iotjs_onexit(void) {
+  remove(JS_DUMP_PATHNAME);
+}
+
+static void iotjs_onsegv(int signum) {
+  remove(JS_DUMP_PATHNAME);
+  signal(signum, SIG_DFL);
+  kill(getpid(), signum);
+}
+
+static void atsignal() {
+  struct sigaction sa_segfault;
```

Comment:
name iotjs_onsegv and sa_segfault are confusing. In fact, you have installed many abnormal signal handler

Reply:
Fixed, thanks :)